### DISCUSSION TITLE
Welcome to moko-geo Discussions!

### DISCUSSION THREAD

**Discussion by Alex009** [icerockdev/moko-geo#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kppv** [comment]
Hello!
Can moko-geo receive location updates after the app is closed?
Specifically, I'm interested in iOS features like `startUpdatingLocation` `startMonitoringSignificantLocationChang`
